sdk/tests/conformance2/reading/read-pixels-pack-parameters.html

```diff
@@ -35,9 +35,18 @@
 </head>
 <body>
 <canvas id="example" width="4" height="4"></canvas>
+<canvas id="canvas2d" width="4" height="4"></canvas>
 <div id="description"></div>
 <div id="console"></div>
 <script>
+"use strict"
+
+var wtu = WebGLTestUtils;
+var expectedColor = [[255, 102, 0, 255],
+                     [2, 200, 102, 255],
+                     [134, 87, 234, 255],
+                     [99, 5, 76, 255]];
```

Comment:
How did you choose these colors? Would it be better to just use more readily visible ones (red, green, blue, yellow) for example?